x/chainlet/module_ibc.go

```diff
@@ -144,11 +124,13 @@ func (im IBCModule) OnChanCloseConfirm(
 // OnRecvPacket implements the IBCModule interface
 func (im IBCModule) OnRecvPacket(
 	ctx sdk.Context,
+	channelVersion string,
 	modulePacket channeltypes.Packet,
 	relayer sdk.AccAddress,
 ) ibcexported.Acknowledgement {
 	var ack channeltypes.Acknowledgement
 
+	//TODO check channel version
```

Comment:
What are these TODOs about? Can we fix them before getting it merged? Otherwise, let's add more context to the comment